python/desc/twinkles/twinklesDBConnections.py

```diff
@@ -38,3 +39,52 @@ class SNCacheDBObj(SNDBObj, CatalogDBObject):
 
     def query_columns(self, *args, **kwargs):
         return CatalogDBObject.query_columns(self, *args, **kwargs)
+
+class TwinklesBulgeObj(GalaxyBulgeObj):
+
+    database = 'LSSTCATSIM'
+    port = 1433
+    host = 'fatboy.phys.washington.edu'
+    driver = 'mssql+pymssql'
+
+    objid = 'galaxyBulge'
+
+class TwinklesDiskObj(GalaxyDiskObj):
+
+    database = 'LSSTCATSIM'
+    port = 1433
+    host = 'fatboy.phys.washington.edu'
+    driver = 'mssql+pymssql'
+
+    objid = 'galaxyDisk'
+
+class TwinklesAgnObj(GalaxyAgnObj):
+
+    database = 'LSSTCATSIM'
+    port = 1433
+    host = 'fatboy.phys.washington.edu'
+    driver = 'mssql+pymssql'
+
+    objid = 'galaxyAgn'
+
+    columns = [('galtileid', None, numpy.int64),
```

Comment:
This `columns` is just the one that is defined in `GalaxyAgnObj`.  There shouldn't be a need to re-define it here.

Is all of this reproduced code a hold-over from Twinkles 1?  Were we running against an out-dated version of `lsst_sims` and needed to re-implement `master` in the Twinkles repo?